terminus-op/values.yaml

```diff
@@ -0,0 +1,47 @@
+replicaCount: 1
```

Comment:
replicaCount should probably be a variable as well. If people choose a storage mechanism that supports ReadWriteMany, then it is possible to have multiple replicas.

Reply:
This is the variable file (Values.yaml) :) The values can always be called from a file!

Reply:
Woops! You are totally right on this one.